frontend/public/components/dashboard/dashboard-card/card.scss

```
@@ -6,14 +6,19 @@
   margin-top: 1.5em;
 }
 
+.co-dashboard-card__button-link {
```

```suggestion
.co-dashboard-card__button-link,
.co-dashboard-card__link {
```

then you don't have the repeat the declarations below